apps/desktop/desktop/src/main/java/bisq/desktop/components/containers/Carousel.java

```diff
@@ -111,6 +105,18 @@ public Carousel(List<Node> items) {
         BorderPane.setAlignment(leftArrowButton, Pos.CENTER);
         BorderPane.setAlignment(rightArrowButton, Pos.CENTER);
 
+        if (!items.isEmpty()) {
+            this.items.addAll(items);
+        }
+    }
+
+    public void initialize() {
+        Rectangle clipRectangle = (Rectangle) itemStackPane.getClip();
```

Comment:
Could be done without cast:

```
if ((itemStackPane.getClip() instanceof Rectangle clipRectangle)) {
            clipRectangle.widthProperty().bind(itemStackPane.widthProperty());
            clipRectangle.heightProperty().bind(itemStackPane.heightProperty());
        }
```

Or even better just make clipRectangle a class field.

Reply:
Fixed with https://github.com/bisq-network/bisq2/pull/3469